DataFormats/TrackCandidate/interface/TrajectoryStopReasons.h

```diff
@@ -22,21 +22,7 @@ enum class StopReason {
 
 // to be kept in synch w/ the above enum ;)
 namespace StopReasonName {
-  static const std::string StopReasonName[] = {
-      "UNINITIALIZED",                 //  0
-      "MAX_HITS",                      //  1
-      "MAX_LOST_HITS",                 //  2
-      "MAX_CONSECUTIVE_LOST_HITS",     //  3
-      "LOST_HIT_FRACTION",             //  4
-      "MIN_PT",                        //  5
-      "CHARGE_SIGNIFICANCE",           //  6
-      "LOOPER",                        //  7
-      "MAX_CCC_LOST_HITS",             //  8
-      "NO_SEGMENTS_FOR_VALID_LAYERS",  //  9
-      "SEED_EXTENSION",                // 10
-      "NOT_STOPPED"                    // 11 (be careful, NOT_STOPPED needs to be the last,
-                                       //     its index differs from the enumeration value)
-  };
+  extern const std::string StopReasonName[];
```

Comment:
How about making this
```suggestion
  extern const std::array<std::string_view, static_cast<size_t>(StopReason::SIZE)>;
```
?

Reply:
Please feel free to make further changes as necessary / wanted. This PR is meant to provide a minimal workaround for https://github.com/root-project/root/issues/13429 which appears to work, up to your decision to merge or wait for a proper fix for the underlying problem in Cling (working on it).

Incidentally, I would argue that this array of names doesn't need any C++ data type, plain `const char *` should be fine. I would still argue that it's better to move the strings into the source files because it means the compiler doesn't have to materialize them in all TUs that include this header, and I think it's not critical for performance.